designer/client/src/components/aiAssistant/components/AssistantMessage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useMessage } from "@assistant-ui/react";
+import { Box, CircularProgress, Typography } from "@mui/material";
+import React, { useMemo } from "react";
+
+import { MarkdownStyled } from "../../graph/node-modal/MarkdownStyled";
+import { ActionsContainer, useHandleActions } from "./actions/ActionsContainer";
+import { CopyContent } from "./actions/CopyContent";
+import { RefreshAssistantAnswer } from "./actions/RefreshAssistantAnswer";
+
+export const AssistantMessage = () => {
+    const { status, content } = useMessage();
+    const { showActions, handleShowActions, handleHideActions } = useHandleActions();
+
+    const messageText = useMemo(() => content.map((part) => part.text).join("\n"), [content]);
+
+    if (status.type === "running") {
```

Comment:
I think `messageText.length === 0` needs to be added here, to not block displaying partial responses.